### PR TITLE
A typo

### PR DESCRIPTION
**Pull request by imbolc**

### CODE REVIEW

File: idioms/coercion-arguments.md [hunk 1]
```diff
@@ -84,8 +84,7 @@ Even ignoring this special example, you may still find that using `&str` will
 give you more flexibility than using a `&String`.
 
 Let's now take an example where someone gives us a sentence, and we want to
-determine if any of the words in the sentence has a word that contains three
-consecutive vowels.
+determine if any of the words in the sentence contains three consecutive vowels.
```
Comment (simonsan) [inline idioms/coercion-arguments.md:87]:
```suggestion
determine if any of the words in the sentence contain three consecutive vowels.
```

File: idioms/coercion-arguments.md [hunk 2]
```diff
@@ -84,8 +84,7 @@ Even ignoring this special example, you may still find that using `&str` will
 give you more flexibility than using a `&String`.
 
 Let's now take an example where someone gives us a sentence, and we want to
-determine if any of the words in the sentence has a word that contains three
-consecutive vowels.
+determine if any of the words in the sentence contain three consecutive vowels.
```
Comment (pickfire) [inline idioms/coercion-arguments.md:87]:
```suggestion
determine if any of the words contains three consecutive vowels.
```

Comment (simonsan) [reply idioms/coercion-arguments.md:87]: I think it's fine like it is now. ;)

**Review by simonsan — APPROVED** [review]

**Comment by simonsan** [comment]
Thank you! :)

**Pull request merged**